modules/ROOT/pages/migrate-client-apps.adoc

```diff
@@ -0,0 +1,148 @@
+= Migrate Client Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: migrate client applications, migrate applications, migrate 
+
+You might have application client credentials residing outside of Anypoint Platform if you were previously using a different API management platform. You must first migrate those client applications to Anypoint Platform, with their client ID and client secret credentials, before you can start using Anypoint Platform.
+
+Because client IDs are unique in Anypoint Platform, you cannot reuse the same client ID in two different accounts. To avoid the duplication, you must specify unique client IDs for your accounts. 
+
+Your client applications can consume APIs whose organization is registered with an Anypoint Platform client provider or an external client provider. The instructions for these two scenarios differ:
+
+* <<migrate_with_internal,Migrate client applications for APIs with Anypoint Platform client providers>>
```

Comment:
It is not `WITH Anypoint Platform client provider` it is `associated to Anypoint Platform client provider`

Reply:
Done! Thanks!